### DISCUSSION TITLE
Is there a way to EXPORT sessions, EDIT them, and then RE-IMPORT the edited version?

### DISCUSSION THREAD

**Discussion by kissingfrogs2003** [sienori/Tab-Session-Manager#1139] [Q&A]
I wondered if this function was available... I think it would be a decent workaround for some of the issues/questions asked around renaming sessions, searching/deleting duplicates, regrouping, etc. Sure it would take work on the user's part BUT for those committed to customized organization it might be a good option for now.

This possibility MAY already exist but I am not familiar enough with coding to know how it would work.

Obviously the options to export and import already exist. I am just not sure how the middle editing part could be possible....especially for users like myself who don't have coding knowledge.

Anyone have ideas?

**Comment by Hjallis** [comment]
Yes, I'm looking for something similar! 
To be able to edit the tabs in the saved sessions? Either directly or as you suggest by exporting, edit and importing again.... 
Thanks!
